owlbot.py

```diff
@@ -1,110 +0,0 @@
-import pathlib
-
-import synthtool as s
-import synthtool.gcp as gcp
-from synthtool.languages import python
-
-
-REPO_ROOT = pathlib.Path(__file__).parent.absolute()
-
-common = gcp.CommonTemplates()
-
-default_version = "v1"
-
-for library in s.get_staging_dirs(default_version):
-    s.replace(
-        [
-            library
-            / "google/cloud/bigquery_reservation_v1/services/reservation_service/client.py",
-            library
-            / "google/cloud/bigquery_reservation_v1/services/reservation_service/async_client.py",
-        ],
-        "assignee=organizations/456``",
-        "assignee=organizations/456``\n",
-    )
-    s.move(library, excludes=["nox.py", "setup.py", "README.rst", "docs/index.rst"])
-
-s.remove_staging_dirs()
-
-# ----------------------------------------------------------------------------
-# Add templated files
-# ----------------------------------------------------------------------------
-templated_files = common.py_library(cov_level=97, microgenerator=True, samples=True,)
-s.move(
-    templated_files,
-    excludes=[".coveragerc"],  # the microgenerator has a good coveragerc file
-)
-
-# Remove the replacements below once
-# https://github.com/googleapis/synthtool/pull/1188 is merged
-
-# Update googleapis/repo-automation-bots repo to main in .kokoro/*.sh files
-s.replace(
-    ".kokoro/*.sh", "repo-automation-bots/tree/master", "repo-automation-bots/tree/main"
-)
-
-# Customize CONTRIBUTING.rst to replace master with main
-s.replace(
-    "CONTRIBUTING.rst",
-    "fetch and merge changes from upstream into master",
-    "fetch and merge changes from upstream into main",
-)
-
-s.replace(
-    "CONTRIBUTING.rst", "git merge upstream/master", "git merge upstream/main",
-)
-
-s.replace(
-    "CONTRIBUTING.rst",
-    """export GOOGLE_CLOUD_TESTING_BRANCH=\"master\"""",
-    """export GOOGLE_CLOUD_TESTING_BRANCH=\"main\"""",
-)
-
-s.replace(
-    "CONTRIBUTING.rst", r"remote \(``master``\)", "remote (``main``)",
-)
-
-s.replace(
-    "CONTRIBUTING.rst", "blob/master/CONTRIBUTING.rst", "blob/main/CONTRIBUTING.rst",
-)
-
-s.replace(
-    "CONTRIBUTING.rst", "blob/master/noxfile.py", "blob/main/noxfile.py",
-)
-
-s.replace(
-    "docs/conf.py", "master_doc", "root_doc",
-)
-
-s.replace(
-    "docs/conf.py", "# The master toctree document.", "# The root toctree document.",
-)
-
-# ----------------------------------------------------------------------------
-# Samples templates
-# ----------------------------------------------------------------------------
-
-python.py_samples()
-
-# ----------------------------------------------------------------------------
-# Final cleanup
-# ----------------------------------------------------------------------------
-
-s.shell.run(["nox", "-s", "blacken"], hide_output=False)
-for noxfile in REPO_ROOT.glob("samples/**/noxfile.py"):
-    s.shell.run(["nox", "-s", "blacken"], cwd=noxfile.parent, hide_output=False)
```

Comment:
I'd like to keep the cleanup of sample code. I don't think the shared templates do this yet, right?

Reply:
This is now part of the shared template for owlbot_main(). See googleapis/synthtool#1276